Mobile Buy SDK/Mobile Buy SDK/Models/BUYObject.h

```diff
@@ -33,7 +34,7 @@
  *
  *  You will generally not need to interact with this class directly.
  */
-@interface BUYObject : NSObject
+@interface BUYObject : NSObject <BUYPersistence>
```

Comment:
`BUYPersistence` is not an appropriate name for this protocol. `BUYPropertyListTransformable` would be better.